infra/examples-dev/aws-google-workspace/variables.tf

```diff
@@ -263,8 +263,8 @@ variable "google_workspace_example_admin" {
 
 variable "salesforce_domain" {
   type        = string
-  default     = ""
   description = "Domain of the Salesforce to connect to (only required if using Salesforce connector). To find your My Domain URL, from Setup, in the Quick Find box, enter My Domain, and then select My Domain"
+  default     = ""
```

Comment:
all these changes are cosmetic, just copy pasted salesforce_domain and msft_tenant_id so they were the same everywhere (type, description, default) in that order